### ISSUE TITLE
How to improve the multi-classifiction performance based on BERT-Base Chinese model?

### ISSUE THREAD

**Issue by NNNNNaaaaaa** [google-research/bert#338]
I fine tune chinese_L-12_H-768_A-12 model  for my classification tasks, where there are 100 classes, the result is not good as expected, the hyperparametes are: learning rate =2e-5,  batch size = 16, epochs=3.

When tuning up the number of epochs to 10, the accuracy on dev data has some increase but is still lower than cnn. If I use larger number of epochs, it shows the training process has been converged. Do you have any suggestion for me? Do I need to try complex layers other than one MLP, or just change the learning rate, or ...?

**Comment by echan00** [comment]
Also interested..

**Comment by ArthurRizar** [comment]
i got a worse result, too

**Comment by zyq001** [comment]
+1
